docs/ethical-advertising.rst

```diff
@@ -0,0 +1,123 @@
+Ethical Advertising
+===================
+
+Read the Docs is a large free website on the internet.
+There is one proven business model to support this kind of site:
```

Comment:
Again, `kind of site` is nondescript.

Also, the voice here -- making the point that read the docs is free, therefore ads -- is solely targeting users wondering why we are displaying ads. This needs to be inclusive to speak towards those landing on the page to learn more about our advertising.